pkg/maps/lbmap/bpfservice_test.go

```diff
@@ -174,3 +174,33 @@ func (b *LBMapTestSuite) TestPrepareUpdate(c *C) {
 	backends = bpfSvc.getBackends()
 	c.Assert(len(backends), Equals, 0)
 }
+
+func (b *LBMapTestSuite) TestGetBackends(c *C) {
```

Comment:
With my patch the unit tests don't fail, why do we need that line?
```patch
index 7cc73f774..eb0220493 100644
--- a/pkg/maps/lbmap/bpfservice.go
+++ b/pkg/maps/lbmap/bpfservice.go
@@ -125,10 +125,8 @@ func (b *bpfService) deleteBackend(backend ServiceValue) {
 
 func (b *bpfService) getBackends() []ServiceValue {
        backends := make([]ServiceValue, len(b.backendsByMapIndex))
-       if len(b.backendsByMapIndex) >= 1 {
-               for i := 1; i <= len(b.backendsByMapIndex); i++ {
-                       backends[i-1] = b.backendsByMapIndex[i].bpfValue
-               }
+       for i := 1; i <= len(b.backendsByMapIndex); i++ {
+               backends[i-1] = b.backendsByMapIndex[i].bpfValue
        }
        return backends
 }
```

Reply:
I've dropped the patch